TaskLayer/XLSearchTask/XLSearchTask.cs

```diff
@@ -143,14 +145,7 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
             proseCreatedWhileRunning.Append("protease = " + CommonParameters.DigestionParams.Protease + "; ");
             proseCreatedWhileRunning.Append("maximum missed cleavages = " + CommonParameters.DigestionParams.MaxMissedCleavages + "; ");
             proseCreatedWhileRunning.Append("minimum peptide length = " + CommonParameters.DigestionParams.MinPeptideLength + "; ");
-            if (CommonParameters.DigestionParams.MaxPeptideLength == null)
-            {
-                proseCreatedWhileRunning.Append("maximum peptide length = unspecified; ");
-            }
-            else
-            {
-                proseCreatedWhileRunning.Append("maximum peptide length = " + CommonParameters.DigestionParams.MaxPeptideLength + "; ");
-            }
+            proseCreatedWhileRunning.Append("maximum peptide length = " + CommonParameters.DigestionParams.MaxPeptideLength + "; ");
```

Comment:
add a check here to see if CommonParameters.DigestionParams.MaxPeptideLength is int.MaxValue; if so, write "unspecified", otherwise write CommonParameters.DigestionParams.MaxPeptideLength